aces/Loris
# [Documentation] Delete duplicate survey_accounts.md

This was a duplicate file of the survey accounts readme.md, stored in the wrong place (help/).

## Brief summary of changes
This is a re-attempted PR (#5207) since my first one was made on the aces repo, rather than my fork.

#### Testing instructions (if applicable)

1. Ensure file has been properly deleted
2. Double check that file still exists in the proper place (survey_accounts/README.md)

#### Links to related tickets (GitHub, Redmine, ...)

Issue #5196
